tests/BinaryOperationTest.php

```diff
@@ -196,6 +196,15 @@ function foo(string $s) : int {
                     '$f' => 'string',
                 ],
             ],
+            'ComplexLiteralBitwise' => [
+                '<?php
+                    /**
+                     * @return 7
+                     */
+                    function scope(){
+                        return 1 | 2 | 4;
```

Comment:
I would suggest to also add a test with `0`, bitwise and (`&`), and brackets.

Reply:
Done